packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -282,6 +273,60 @@ export default {
          * focus on the input field.
          */
 
+        describeFirstnameErrorMessage () {
+            if (this.$v.firstName.$dirty) {
+                if (!this.$v.firstName.required) {
+                    return 'Please include your first name';
+                }
+                if (!this.$v.firstName.maxLength) {
+                    return 'First name exceeds 50 characters';
+                }
+                if (!this.$v.firstName.meetsCharacterValidationRules) {
+                    return 'Your name can only contain letters, hyphens or apostrophes';
```

Comment:
I would be careful with such rules. Is it documented somewhere?

Reply:
For these the code is the documentation, I actually think this is correct as they're not complicated and they're described by the tests.

Reply:
I mean there're probably names out there that contain numbers, etc, for example, X Æ A-12 😂.